src/test/java/TVTest.java

```diff
@@ -0,0 +1,191 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TVTest {
+
+    @Test
+    void toggleOnOff() {
+        TV tv = new TV();  //tv not on
```

Comment:
Instantiate the TV in a setUp() method prior?

Reply:
Yes, originally I was going to write tests with multiple tvs, but ended up going with just one that should be in the setup.
I'll add a test that does create a few tvs just to show why these need to be instance variables.